src/DistributedLock.Redis/Primitives/RedisMutexPrimitive.cs

```diff
@@ -20,11 +20,11 @@ public RedisMutexPrimitive(RedisKey key, RedisValue lockId, RedLockTimeouts time
     public TimeoutValue AcquireTimeout => this._timeouts.AcquireTimeout;
 
     private static readonly RedisScript<RedisMutexPrimitive> ReleaseScript = new(@"
-        if redis.call('get', @key) == @lockId then
-            return redis.call('del', @key)
+        if redis.call('get', KEYS[1]) == ARGV[1] then
+            return redis.call('del', KEYS[1])
         end
         return 0",
-        p => new { key = p._key, lockId = p._lockId }
+        p => ([p._key], [p._lockId])
```

Comment:
This is changing a lot more code than is necessary. LuaScript is already supposed to take any parameters of type RedisKey and send them as KEYS. Instead, let's go through and make sure all keys are properly typed.